### PR TITLE
POL-352-Google-Schedule-Instance

### PR DESCRIPTION
**Pull request by KVivek03**
### Description

POL-352-Google-Schedule-Instance

### Issues Resolved

[List any existing issues this PR resolves]

### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable
- [ ] New functionality has been documented in CHANGELOG.MD

### CODE REVIEW

File: cost/google/schedule_instance/README.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Google Schedule Instance
+
+## What it does
+
+This Policy Template allows you to schedule start and stop times for your instance, along with the option to terminate instance, update and delete schedule.
```
Comment (cdwilhelm) [inline cost/google/schedule_instance/README.md:5]: I think this area should include the cloud the policy supports.  I know the Heading includes it, but this should include Google also.  Same for the short_description on the policy_template

File: cost/google/schedule_instance/README.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+This Policy Template allows you to schedule start and stop times for your instance, along with the option to terminate instance, update and delete schedule.         
+
+## How to Use
+
+This policy relies on a label with format 'schedule' to stop and start instances based on a schedule. The label value defines the schedule with a start time(start hour and start minute), stop time(stop hour and stop minute), days of the week and timezone. The start and stop time are in 24 hour format, and the days of the week are two character abbreviation for example: mo, tu, we. See full example below.. Use a Timezone TZ value to indicate a timezone to stop/start the instance(s) 
+
+## schedule Label Example
+
+Since label supports only `-`, `_`, lowercase characters, numbers and International characters, We are replacing `:` with `m` in start and stop time, and ASCII code for special character `/` and `+` in timezone, surrounded by `_`.
```
Comment (cdwilhelm) [inline cost/google/schedule_instance/README.md:13]: I'm not excited about the use of ascii code for special characters.   See notes in JIRA.

File: cost/google/schedule_instance/google_schedule_instance.pt [hunk 3]
```diff
@@ -0,0 +1,638 @@
...
+  $labelFingerprint   = $body['labelFingerprint']
+  $labels_obj= $body['labels']
+end
+
+define get_tzlist() return $timezones do
+  $params = {
+    verb: 'get',
+    host: 'raw.githubusercontent.com',
+    https: true,
+    href: '/rightscale/policy_templates/POL-352-Google-Schedule-Instance/data/tz_database/timezones_list.json',
```
Comment (cdwilhelm) [inline cost/google/schedule_instance/google_schedule_instance.pt:612]: This should probably be in the master branch

Comment (KVivek03) [reply cost/google/schedule_instance/google_schedule_instance.pt:612]: modified

**Pull request merged**